tests/lib/nrf_modem_lib/trace_backends/uart/src/main.c

```diff
@@ -0,0 +1,294 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdbool.h>
+#include <string.h>
+
+#include <zephyr/kernel.h>
+
+#include <unity.h>
+
+#include "trace_backend.h"
+
+#include "mock_nrfx_uarte.h"
+#include "mock_pinctrl.h"
```

Comment:
lots of whitespace here

Reply:
It's on purpose to group the headers

Reply:
I usually group them as public includes and private includes, e.g.
```
#include <>

#include ""
```
and in each group I order them according to my own preference, something like standard headers, zephyr headers, ncs headers etc. I think that works a bit better because it avoids cases like this, where each header is seen as its own group  and we end up with 5 groups for 7 headers, and a lot of whitespace.